reporter/symbol_uploader.go

```diff
@@ -162,6 +165,19 @@ func (d *DatadogSymbolUploader) Start(ctx context.Context) {
 		symbolQueryMaxBatchSize = 1
 	}
 
+	memoryBudget, err := cgroup.GetMemoryBudget()
+	// Couldn't read the budget from cgroups
+	if err != nil {
+		log.Warnf("Failed to fetch cgroup memory limit: %v", err)
+	}
+
+	if memoryBudget == -1 {
+		log.Info("No memory limit found in cgroup, unlimited budget for symbol upload")
+		memoryBudget = math.MaxInt64
+	} else {
+		log.Infof("Memory budget for symbol upload of %v", memoryBudget)
```

Comment:
Nit: I'd switch this to debug. While this is useful for us to know, I don't think it's generally useful information for the users.